build.rb

```diff
@@ -120,10 +134,7 @@ def name_object(var_prefix, language_map)
   output_filename = legislative_dir.join(legislature_h[:house_item_id], 'popolo-m17n.json')
```

Comment:
a little bit nit-like, but if I'm following the logic here, this will actually be a Pathname, rather than a filename. That's not massively important except it means that when it comes to writing to this later, you can call `.write` directly on a Pathname, which is a little disguised by giving it a slightly misleading name here.